staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness.go

```diff
@@ -111,7 +111,12 @@ func WithPriorityAndFairness(
 			httplog.AddKeyValue(ctx, "apf_pl", truncateLogField(pl.Name))
 			httplog.AddKeyValue(ctx, "apf_fs", truncateLogField(fs.Name))
 			httplog.AddKeyValue(ctx, "apf_fd", truncateLogField(flowDistinguisher))
-			return workEstimator(r, fs.Name, pl.Name)
+
+			workEstimate := workEstimator(r, fs.Name, pl.Name)
+			httplog.AddKeyValue(ctx, "apf_i_seats", workEstimate.InitialSeats)
+			httplog.AddKeyValue(ctx, "apf_f_seatss", workEstimate.FinalSeats)
```

Comment:
Now that #105873 has merged, we might not need to add this to every httplog line.

Reply:
yeah, this is for here only, i already have a tool setup with Loki where I use the seats from httplog.

Reply:
Metrics doesn't solve this problem - metrics show us patterns and let us diagnose the higher-level problems, but metrics will never let us dig deep into individual requests, which is where those logs are helpful.